core/src/apps/common/safety_checks.py

```diff
@@ -0,0 +1,56 @@
+import storage.device
+from trezor.messages import SafetyCheckLevel
+
+if False:
+    from typing import Optional, Tuple
+    from trezor.messages.ApplySettings import EnumTypeSafetyCheckLevel
+
+_temporary_safety_checks = None  # type: Optional[EnumTypeSafetyCheckLevel]
```

Comment:
this must be part of the session-less cache; otherwise we risk losing the value if `apps.common.safety_checks` are ever unimported